addons/account/wizard/account_resequence.py

```diff
@@ -30,6 +30,8 @@ def default_get(self, fields_list):
             active_move_ids = self.env['account.move'].browse(self.env.context['active_ids'])
         if len(active_move_ids.journal_id) > 1:
             raise UserError(_('You can only resequence items from the same journal'))
+        if active_move_ids.filtered("inalterable_hash"):
+            raise UserError(_('Some or all of the entries are secured and cannot be modified'))
```

Comment:
@FlorianGilbert, could you review this please? I'm not sure when the restriction should be applied and  which check should be done

Reply:
Hey, I will talk with my PO about that and I come back to you soon!

Reply:
(May be we want to add this check only for l10n_fr)

Reply:
@yelizariev Hello, you have to add this only for l10_fr, the law in France prohibits the modification of invoice numbers.

Reply:
This feature is mainly used by l10n_fr, so it's kind of the same. But I'm ok to add it to 10n_fr only

Reply:
@yelizariev @tsb-odoo doesn't want that so, before doing anything ask him please :smile: